crest/Assets/Crest/Crest/Scripts/LodData/SplinePointData/SplinePointDataNone.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Crest Ocean System
+
+// This file is subject to the MIT License as seen in the root of this folder structure (LICENSE)
+
+using Crest.Spline;
+using UnityEngine;
+
+namespace Crest
+{
+    /// <summary>
+    /// No data. This should not be attached to any apline point, but is used as a symbol
+    /// in the code when no data is required.
+    /// </summary>
+    public class SplinePointDataNone : MonoBehaviour, ISplinePointCustomData
```

Comment:
Adding `[AddComponentMenu("")]` will hide it from the menu (I tried class visibility modifiers but they don't change anything). We should consider whether it is worthwhile adding it to the others too.

Reply:
interesting, did not know that. done. i guess it never needs to be added manually, if everything else is working